### DISCUSSION TITLE
next build just sits at "Collecting page data..."

### DISCUSSION THREAD

**Discussion by chrisabrams** [vercel/next.js#20907] [Help]
Trying to build an app with Next.js, the build just sits at "Collecting page data..." forever. I've tried restarting, removing cache, nothing makes the build move forward. Is this a known issue? Any way to resolve?

I've tried Next versions 9.5.2, 10.0.4, and 10.0.5; all same results.

**Comment by leerob** [comment]
Hello! Please provide a reproducible example (preferable a CodeSandbox link) so we can verify this is an issue with Next.js. This will allow us to inspect the `next build` output.
